projects/core/src/util/regex-pattern.ts

```diff
@@ -19,3 +19,8 @@ export const MIN_ONE_SPECIAL_CHARACTER_PATTERN =
   /^(?=.*?[!@#$%^*()_\-+{};:.,]).*$/;
 
 export const MIN_SIX_CHARACTERS_PATTERN = /^.{6,}$/;
+
+export const CONSECUTIVE_CHARACTERS = /(.)\1+/;
+
+export const STRONG_PASSWORD_PATTERN =
+  /^(?!.*(.)\1)(?=.*?[A-Z])(?=.*?\d)(?=.*?[!@#$%^*()_\-+{};:.,]).{6,}$/;
```

Comment:
By checking this regex
No two consecutive identical characters.
At least one uppercase letter.
At least one digit.
At least one special character from the allowed set.
Minimum length of 6 characters

Does this check match the backend validation? As far as I understand, the validation should match the backend validation.

Limited special characters: Only a small set of special characters is allowed. (?=.*?[!@#$%^*()_\-+{};:.,]). is this correct?

Reply:
I'm not sure if the validation should align with the backend requirements. If you take a look at the ticket comments we agreed that the backend would do the proper validation and the frontend will only validate simple checks for UX reasons.

However, I don't believe the backend has been updated to validate the new rule - > consecutive character

Limited special characters -> tbh I haven't touched it, what are you proposing?

Reply:
Limited special characters is a pattern from line 11, so I believe we should be fine.  thanks